com.epam.zoo/src/Elephant.java

```diff
@@ -1,15 +1,15 @@
-public class Elephant extends Animal {
-    public Elephant(String name, ZooType zooType) {
-        super(name, "grass+fruit", 12);
-    }
-
-    @Override
-    public String getEnclosure() {
-        return "Elephant Cage";
-    }
-
-    @Override
-    public String getCageType() {
-        return "";
-    }
-}
+public class Elephant extends Animal {
+    public Elephant(String name, ZooType zooType) {
+        super(name, "grass+fruit", 12);
```

Comment:
As I see **food** field should be enum - not string

In animal class:
protected List<FoodType> foodTypes;